modeler.js

```diff
@@ -0,0 +1,131 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE
+};
+
+// An action that creates a new project.
+async function createProject([projectName]) {
+  // Every request needs an access token.
+  const accessToken = await getAccessToken(authorizationConfiguration);
+
+  // These settings come from your .env file.
+  const modelerApiUrl = process.env.MODELER_API_URL;
+
+  // This is the API endpoint to create a new project.
+  const url = `${modelerApiUrl}/projects`;
+
+  // Configure the API call.
+  const options = {
+    method: "POST",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    },
+    data: {
+      // The body contains information about the new project.
+      projectName: projectName
+    }
+  };
+
+  try {
+    // Call the add endpoint.
+    const response = await axios(options);
+
+    // Process the results from the API call.
+    const newProject = response.data;
+
+    console.log(
+      `Project added! Name: ${newProject.name}. ID: ${newProject.projectId}.`
+    );
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+}
+
+// An action that views one project.
+async function viewProject([projectId]) {
+  // Every request needs an access token.
+  const accessToken = await getAccessToken(authorizationConfiguration);
+
+  // These settings come from your .env file.
+  const modelerApiUrl = process.env.MODELER_API_URL;
+
+  // This is the API endpoint to view a single project.
+  const url = `${modelerApiUrl}/projects/${projectId}`;
+
+  // Configure the API call.
+  const options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+
+  try {
+    const response = await axios(options);
+
+    // Process the results from the API call.
+    const id = response.data;
+
+    // Emit the project details.
+    console.log("Project:", id);
```

Comment:
```suggestion
    const data = response.data;

    // Emit the project details.
    console.log("Project:", data);
```

This is a more accurate name for this variable.